<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1317</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>1121</width>
      <height>541</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;panier&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;panier&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 127);
</string>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ajouter c</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>date_livraison</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>119</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>fournisseur</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="montant_total">
      <property name="geometry">
       <rect>
        <x>592</x>
        <y>110</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="liste_article">
      <property name="geometry">
       <rect>
        <x>592</x>
        <y>60</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>109</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>montant_total</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>169</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>date_commande</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>169</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>etat_commande</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>49</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>liste_article</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fournisseur">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>320</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_commande">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">alternate-background-color: rgb(170, 0, 0);
background-color: rgb(170, 0, 0);
border-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>id_commande</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_commande">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>110</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_livraison">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>110</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_etat_commande">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>170</y>
        <width>101</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(167, 167, 167);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mettre_a_jour">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>320</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>mise à jour</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>afficher c</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>731</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="afficher">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>440</y>
        <width>171</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>60</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>par défaut</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ordre croissant montant total</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ordre décroissant montant total</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ordre croissant id commande</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ordre décroissant id commande</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ordre alphabétique état commande</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ordre croissant date commande</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ordre décroissant date commande</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_recherche">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="recherche">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>recherche</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>supprimer c</string>
     </attribute>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>230</y>
        <width>241</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>id_commande</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; font-weight:700; font-style:italic; text-decoration: underline; color:#aa0000; vertical-align:super;&quot;&gt;choisir la commande pour supprimer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_commande_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>140</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>mise à jour c</string>
     </attribute>
     <widget class="QPushButton" name="statistique">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>291</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>statistique</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>511</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>40</y>
        <width>501</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="ajouterArticle">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>171</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>ajouterArticle</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viderPanier">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>171</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(90, 90, 90);
</string>
      </property>
      <property name="text">
       <string>viderPanier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="afficherPanier">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>230</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>afficherPanier</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textEdit_panier">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>381</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(220, 220, 220);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>panier</string>
      </property>
     </widget>
     <widget class="QLabel" name="lineEdit_article1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>article</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_article">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prix">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>113</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lineEdit_prix3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>prix</string>
      </property>
     </widget>
     <widget class="QPushButton" name="calculerPrixTotal">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>350</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>calculerPrixTotal</string>
      </property>
     </widget>
     <widget class="QLabel" name="prix">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>339</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimerArticleButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>171</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(90, 90, 90);</string>
      </property>
      <property name="text">
       <string>supprimerArticleButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_alert">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>400</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>181</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
background-color: rgb(29, 29, 29);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>130</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 148, 148);</string>
    </property>
    <property name="text">
     <string>Tableau de bord</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 148, 148);
background-color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>ajouter commande</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>230</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 148, 148);</string>
    </property>
    <property name="text">
     <string>gestion de stock</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>280</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 148, 148);</string>
    </property>
    <property name="text">
     <string>ajouter technicien</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>330</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 148, 148);</string>
    </property>
    <property name="text">
     <string>ajouter client</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 148, 148);</string>
    </property>
    <property name="text">
     <string>gestion financiere</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1317</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
